tests/unit/iontests.js

```diff
@@ -344,6 +344,16 @@ let goodSkipList = toSkipList([
     'ion-tests/iontestdata/good/symbolZero.ion',
     'ion-tests/iontestdata/good/utf16.ion',
     'ion-tests/iontestdata/good/utf32.ion',
+
+    // Support for reading LongInts was removed as part of: https://github.com/amzn/ion-js/issues/310
```

Comment:
I've been running a script to regenerate these skip lists on a weekly basis, and that process will drop comments like this.  Maybe we need to rethink that approach, or find a different way to maintain comments such as this...?

Reply:
It's valuable to have a documented mapping from skiplist entries to github issues. It helps contributors to know when their patch has fully addressed the (known aspects of the) problem they're tackling.

I'm all for keeping that mapping in a github meta issue if you prefer. We could add a link to the issue to the top of the script-generated skiplist.

Reply:
We'll figure out something; perhaps it's time to stop auto-generating the skiplist.